src/features/item-details/__tests__/ItemProperties.test.js

```diff
@@ -73,4 +81,57 @@ describe('features/item-details/ItemProperties', () => {
 
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should render archived date when feature is enabled', () => {
+        const wrapper = getWrapper({
+            archivedAt: '2014-12-12T11:04:26-08:00',
+            features: {
+                details: {
+                    archivedAt: {
+                        enabled: true,
+                    },
+                },
+            },
+        });
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should render empty archived date when feature is enabled', () => {
+        const wrapper = getWrapper({
+            archivedAt: null,
+            features: {
+                details: {
+                    archivedAt: {
+                        enabled: true,
+                    },
+                },
+            },
+        });
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should not render archived date when feature is disabled', () => {
+        const wrapper = getWrapper({
+            archivedAt: null,
+            features: {
+                details: {
+                    archivedAt: {
+                        enabled: false,
+                    },
+                },
+            },
+        });
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should not render archived date when feature is not set', () => {
+        const wrapper = getWrapper({
+            archivedAt: null,
```

Comment:
I think we should actually supply a `archivedAt` here so we are certain that the tests are passing because the feature is off or undefined. from reading the test itself, we dont know if its not rendering because archivedAt is null or feature is null/undefined.

Reply:
Changed